lib/x11utils.pm

```diff
@@ -322,4 +324,34 @@ sub untick_welcome_on_next_startup {
     }
 }
 
+=head2 workaround_boo1170586
+
+ workaround_boo1170586();
+
+Kill broken opensuse-welcome window and restart it properly to workaround boo#1170586.
+
+=cut
+sub workaround_boo1170586 {
+    record_soft_failure('opensuse-welcome auto-start is broken, boo#1170586');
+    x11_start_program('killall /usr/bin/opensuse-welcome', target_match => 'generic-desktop');
```

Comment:
That's a good idea but I think the needle https://openqa.opensuse.org/tests/1259161#step/opensuse_welcome/7 should be deleted now to actually not match anymore

Reply:
This is needed for `first_boot`: https://openqa.opensuse.org/tests/1259161#step/first_boot/5